.circleci/config.yml

```
@@ -118,14 +118,124 @@ jobs:
+                                | [., . / "."]
+                                # ["2.2.0-snap1", ["2", "2", "0-snap1"]]
+                                # select is filter
+                                # we only keep versions after 1.x line
+                                | select(
+                                          # first element of second element: major version number
+                                          # converted to number so we can make a numeric comparison
+                                         (.[1][0] | tonumber) > 1
+                                        )
+                                # we keep just the version number, we do not need the vector of
+                                # elements anymore
+                                | .[0]
+                                ')
+             graphql() (
+               version=$1
+               echo '
+               {
+                 repository (name: "daml", owner: "digital-asset") {
+                   release (tagName: "'$version'") {
+                     releaseAssets (first: 100) {
+                       nodes {
+                         name
+                       }
+                     }
+                   }
+                 }
+               }
+               '
+             )
+
+             for version in $snapshots; do
+                 echo "$version"
+                 if curl -Ifs https://docs.daml.com/$version/index.html &> /dev/null
+                 then echo "  Nothing to do."
+                 else
+                     echo "  Getting Canton version"
+                     canton=$(curl -H "Authorization: bearer $GITHUB_TOKEN" \
+                                   --fail \
+                                   --silent \
+                                   https://api.github.com/graphql \
+                                   -XPOST \
+                                   -d"{\"query\":\"$(graphql $version)}" \
+                            | jq -r '.data.repository.release.releaseAssets.nodes[].name' \
+                            | grep canton \
+                            | head -1 \
+                            | sed 's/canton-open-source-\(.*\)\.tar\.gz/\1/')
+                     echo "  Building $version with $canton"
+                     jq -n \
+                        --arg daml $version \
+                        --arg canton $canton \
+                        '{daml: $daml, canton: $canton, prefix: $daml}' \
+                        > LATEST
+
+                     docs/download.sh
+                     docs/build.sh
+                     upload=$(mktemp -d)
+                     tar xf docs/workdir/target/html-docs-$version.tar.gz -C $upload --strip-components=1
+                     echo aws s3 cp $upload s3://docs-daml-com/$version --recursive --acl public-read --region us-east-1
+                     echo aws cloudfront create-invalidation --distribution-id E1U753I56ERH55 --paths "/$version/*"
```

```suggestion
                     aws s3 cp $upload s3://docs-daml-com/$version --recursive --acl public-read --region us-east-1
                     aws cloudfront create-invalidation --distribution-id E1U753I56ERH55 --paths "/$version/*"
```
After review, the `echo` prefix should be removed.